sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/coverage/CxxCoverageSensor.java

```diff
@@ -179,7 +179,7 @@ private void saveMeasures(SensorContext context,
           }            
         }
       } else {
-        CxxUtils.LOG.warn("Cannot find the file '{}', ignoring coverage measures", filePath);
+        CxxUtils.LOG.trace("Cannot find the file '{}', ignoring coverage measures", filePath);
```

Comment:
these ones for me were printing loads of files outside project because coverage is collected for also files outside project. the platform should handle the proper importing of files inside project and we should not bother for files outside. so this is redudant, ive moved to trace since trace does not print in normal or debug mode